proto/src/determined/project/v1/project.proto

```diff
@@ -10,6 +10,28 @@ import "google/protobuf/timestamp.proto";
 import "google/protobuf/wrappers.proto";
 import "protoc-gen-swagger/options/annotations.proto";
 
+// GeneralColumn is the pre-defined column names for experiment list table.
+enum GeneralColumn {
```

Comment:
I am not sure this enum is worth it. Maybe we could have simple strings, just like hyperparameters and metrics? 
- it'd make the API look more uniform
- we can validate if the strings are in an allowlist of columns in go code instead

Reply:
I added these in proto because I want this to be a shared information between front end and back end, so front end would know this information before API response. I wonder what would be the disadvantage against using Enum?

Reply:
with enums, query parameters will look like `?columns=GENERAL_COLUMN_ID&columns=GENERAL_COLUMN_NAME&hyperparameters=dropout1&hyperparameters=lr&metrics=training.loss`
without, `?columns=id&columns=name&hyperparameters=dropout1&hyperparameters=lr&metrics=training.loss`